### PR TITLE
xmlb-3 and xmlb-4 are recreated in SVG format

### PR DESCRIPTION
**Pull request by DmitriGit**
xmlb-3 and xmlb-4 are recreated in SVG format

### CODE REVIEW

File: spec/src/main/asciidoc/images/xmlb-3.svg [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+			<text x="4" y="289.04" class="st6" v:langID="1040"><v:paragraph/><v:tabList/>Portable<v:newlineChar/><tspan x="4"
+						dy="1.2em" class="st11">JAXB</tspan><tspan class="st11">-</tspan><tspan class="st11">annotated<v:newlineChar/></tspan><tspan
+						x="4" dy="1.2em" class="st11">classes</tspan></text>		</g>
+		<g id="shape66-81" v:mID="66" v:groupContext="shape" transform="translate(346.547,-193.071)">
+			<title>Foglio.66</title>
+			<rect x="0" y="278.83" width="96.378" height="34.4207" class="st3"/>
+		</g>
+		<g id="shape67-83" v:mID="67" v:groupContext="shape" transform="translate(347.462,-193.273)">
+			<title>Foglio.67</title>
+			<desc>Package Javax.xml.bind</desc>
```
Comment (lukasj) [inline spec/src/main/asciidoc/images/xmlb-3.svg:184]: package name should be all lowercase

File: spec/src/main/asciidoc/images/xmlb-4.svg [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+			<rect x="0" y="202.7" width="79.158" height="34.0157" class="st4"/>
+			<text x="4" y="212.51" class="st7" v:langID="1040"><v:paragraph/><v:tabList/>Schema Derived <tspan x="4" dy="1.2em"
+						class="st6">Interfaces</tspan>,<v:newlineChar/><tspan x="4" dy="1.2em" class="st6">Factory Methods</tspan></text>		</g>
+		<g id="shape66-57" v:mID="66" v:groupContext="shape" transform="translate(346.83,-122.918)">
+			<title>Foglio.66</title>
+			<rect x="0" y="163.632" width="96.378" height="73.084" class="st3"/>
+		</g>
+		<g id="shape67-59" v:mID="67" v:groupContext="shape" transform="translate(349.163,-150.956)">
+			<title>Foglio.67</title>
+			<desc>Package Javax.xml.bind</desc>
```
Comment (lukasj) [inline spec/src/main/asciidoc/images/xmlb-4.svg:137]: package name should be all lowercase

Comment (DmitriGit) [reply spec/src/main/asciidoc/images/xmlb-4.svg:137]: Yes, you are right! I correct it right now.

Comment (DmitriGit) [reply spec/src/main/asciidoc/images/xmlb-4.svg]: Now package name is in lowercase.
Sorry for typo and thank you!
Dmitri

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**